test/extensions/filters/http/cors/cors_filter_test.cc

```diff
@@ -25,10 +26,13 @@ namespace Cors {
 
 class CorsFilterTest : public testing::Test {
 public:
-  CorsFilterTest() : filter_() {
```

Comment:
up to you, but as an aside, you can avoid many of the changes in this file by still using initializer list init. Something like this:

```
CorsFilterTest() : config_(new CorsFilterConfig("test.", stats_), filter_(config_) 
```

And move `stats_` definition above `config_`